README.rst

```diff
@@ -1,66 +1,202 @@
-Welcome to drf_polymorphic's documentation!
-=================================================
+drf_polymorphic
+===============
 
 :Version: 0.1.0
 :Source: https://github.com/maykinmedia/drf_polymorphic
 :Keywords: django, rest, polymorphic
-:PythonVersion: 3.7
+:PythonVersion: 3.7, 3.8
 
-|build-status| |coverage| |black|
+|build-status| |coverage| |linting|
```

Comment:
Take a look at the https://github.com/maykinmedia/zgw-consumers badges as well - there's PYPI meta information for the Python versions, Django versions & version of package on PyPI